packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1684,6 +1811,21 @@ export class Combobox
       <div ariaHidden="true" class={CSS.floatingUIContainer} ref={setFloatingEl}>
         <div class={classes} ref={setContainerEl}>
           <ul class={{ list: true, "list--hide": !open }}>
+            {this.selectAllEnabled &&
+              this.selectionMode !== "single" &&
+              this.selectionMode !== "single-persist" && (
+                <calcite-combobox-item
+                  ariaLabel={this.messages.selectAll}
+                  class={CSS.selectAll}
+                  id={`${this.guid}-select-all-enabled`}
+                  ref={this.setSelectAllComboboxItemReferenceEl}
+                  role="option"
```

Comment:
Why is the role being set here? Shouldn't the component be setting the role? It seems like this would be the responsibility of the component itself.